ui-backend/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/compose/ComposeApplication.java

```diff
@@ -321,7 +321,7 @@ public int compare(Map.Entry<String, Object> entry1, Map.Entry<String, Object> e
   public void init() {
 
     get(
-        "/compose/startup",
+        "/internal/compose/startup",
```

Comment:
So the routes now require this, but only because of how jaxrs servers are registered in the new OSGI whiteboard web services model. There is a plan to replace those with spark servlets which can have their context configured. We won't need to rewrite all of these - so leaving it here for now, but after merging with work @glenhein has been doing we won't need it.